.changeset/curvy-tools-notice.md

```diff
@@ -4,4 +4,5 @@
 ---
 
 - Add new placeholder text variable.
-- Change input & select placeholder to use the new variable, also change font weight to light.
+- Change input & select placeholder to use the new variable & set font weight to light.
```

Comment:
can we separate the changesets? one for variables (minor), the other for css and react (patch).

```md
---
'@itwin/itwinui-variables': minor
---

Added new `--iui-color-text-placeholder` variable.
```
```md
---
'@itwin/itwinui-css': patch
'@itwin/itwinui-react': patch
---

Improved color contrast for input and select placeholder text.
```

Reply:
Done, but just so I know what is the benefit / reason for doing so?

Reply:
multiple reasons:
- the only new api is the new variable, so only itwinui-variables needs a `minor` bump
- the new variable is an implementation detail of css/react components, so doesn't necessarily need to be mentioned in the css/react changelogs
- the component changes are not relevant for variables changelog